### PR TITLE
#112-show related pages at bottom of current page

### LINKED ISSUE #112: Create Afla mai multe section for content pages
long story short: unele dintre paginile de continut simplu (nu cele gen ai simptome) vor fi alcatuite din subpagini si apare meniul din sidebar (care are doar varianta unfolded). cand dai click de sus pe una din pagini, iti apare la content doar prima subpagina (daca are subpagini) sau toata pagina (daca nu are subpagini) **si apare si bucata cu "afla mai multe" daca mai are si alte subpagini.**

Figma here: https://www.figma.com/proto/XDcxBzr6MdMlrZmmlBTYvu/Content-design---CeMaFac?node-id=1%3A214&scaling=min-zoom

### PR DESCRIPTION
**Pull request by bogdan00**
Closes #112 

image

### CODE REVIEW

**Comment by aniri** [comment]
@bogdan00 looks awesome :rocket: would it be possible to also auto scroll back to the top of the content when clicking on an item?

**Comment by bogdan00** [comment]
sure

File: frontend/src/components/ContentPage/index.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
   SocialsShare
 } from "@code4ro/taskforce-fe-components";
 
 function ContentPage({ page, subPage }) {
+  const history = useHistory();
+  const navigate = slug => {
+    // Fix SecurityError of pushState on History
+    const cleanPageSlug = `/${page.slug}/${slug}`.replace(/\/+$/, "");
+    const navTarget = page.slug !== cleanPageSlug ? cleanPageSlug : page.slug;
+    history.push(navTarget);
```
Comment (moonflare) [inline frontend/src/components/ContentPage/index.js]: nit: Isn't this a bit redundant? I think we could always navigate the clean slug
```suggestion
    const cleanPageSlug = `/${page.slug}/${slug}`.replace(/\/+$/, "");
    history.push(cleanPageSlug);
```

File: frontend/src/components/ContentPage/index.js [hunk 2]
```diff
@@ -20,11 +31,37 @@ function ContentPage({ page, subPage }) {
...
+    }
+    const items = page.content
+      .filter(item => item.slug !== subPage.slug)
+      .map(item => (
+        <ListItem
+          key={item.display_order}
+          title={item.title}
+          onClick={() => {
+            navigate(item.slug);
+            scrollAnchorRef.current.scrollIntoView(true);
```
Comment (moonflare) [inline frontend/src/components/ContentPage/index.js]: Can we please add `behavior: "smooth"` to the `scrollIntoView`? 🙏

Comment (bogdan00) [reply frontend/src/components/ContentPage/index.js]: as per https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView (& compatibility chart) im inserting smooth scroll to top as a method that needs a fallback

**Pull request merged**